README.md

```diff
@@ -1 +1,141 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Introduction
+
+The parliament challenge by Trustly is approached as follows:
+
+1. Get speeches resource from its respective apis. Since speeches data is in a nested dict, the for loop has been used to iterate over the required dict which is "anforande". The speeches api allows for extracting the ten latest speeches.
+
+> Speech api: http://data.riksdagen.se/anforandelista/?anftyp=Nej&sz=10&utformat=json
+
+2. Get members resource from its respective apis. Since members data is in a nested dict, the for loop has been used to iterate over the required dict which is "person". All members data has been extracted.
```

Comment:
Is it really nested? How?

Reply:
the resource is as follows:

{
  "personlista": {
    "@systemdatum": "2019-12-02 16:51:13",
    "@hits": "349",
    "person": [
      {{"keys": "values",
        "keys": "values"
}]

so it becomes nested